internal/business/business.go

```diff
@@ -104,10 +104,12 @@ func publicMain(ctx context.Context, cfg *config.Config) error {
 		return fmt.Errorf("creating audit logger: %w", err)
 	}
 
+	pkceSource := pkce.Source{}
 	sessionManager := session.NewManager(
 		oidcProviderRepo,
 		sessionRepo,
 		auditLogger,
+		pkceSource,
```

Comment:
It isn't necessarily since `pkce.Source` was used by value, not by pointer. It's better to avoid irrelevant changes in the PR.